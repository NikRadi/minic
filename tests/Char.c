int add3(char x, char y, char z) {
    return x + y + z;
}

int main() {
    char x1 = 11;
    char x2 = 22;
    printf("%d\n", x1);
    printf("%d\n", x2);
    printf("%d\n", sizeof(x1));

    char x3 = add3(1, 2, 3);
    printf("%d\n", x3);
}
