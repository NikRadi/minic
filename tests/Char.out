11
22
1
6